added support for conda environments.

gregorgebhardt opened the pull request.

bobthecow commented:
Conda is an alternative to virtualenv, right? Is there ever a reason to use them together?

For the rubies, bobthefish has one segment that supports several different version mangers and bundlers, since they're generally mutually exclusive. Would it make sense to do the same with pythons?

gregorgebhardt commented:
I think, you're right. At least, I've never used them together and don't know why this should be necessary. I removed the conda-segment and added the functionality to the virtualenv function.

bobthecow commented:
Thank you!
